File: src/main/resources/com/zkdas/oop/service/ValueValidator.cs
using System;

namespace OOP.service
{   
    /// <summary>
    ///  валидатор проверки длины поля
    /// </summary>
    public class ValueValidator
    {
        public static void AssertStringOnLength(string value, int maxLength, string propertyName)
        {
            if(value.Length > maxLength)
            {
                throw new Exception($"'{propertyName}' must be less than {maxLength} characters.");
            }
        }
    }
}

File: src/main/java/com/zkdas/oop/service/ValueValidator.java
package com.zkdas.oop.service;

/// <summary>
///  валидатор проверки длины поля
/// </summary>
public class ValueValidator {
    public static void AssertStringOnLength(String value, int maxLength, String propertyName) throws Exception {
        if (value.length() > maxLength) {
            throw new Exception(propertyName + " должна быть короче " + maxLength + "символов.");
        }
    }
}
